modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/CreateDumpFutureTask.java

```diff
@@ -446,7 +452,7 @@ public PartitionDumpContext(CacheGroupContext gctx, int part, ConcurrentMap<Long
 
                 File dumpFile = new File(groupDirectory(gctx), PART_FILE_PREFIX + part + DUMP_FILE_EXT);
```

Comment:
We can calculate correct file name for compress mode here. And created file only in one place.

Reply:
Please, double check - do we really need to create file explicitly.
As far as I can see - FileIOFactory must create new file if it not exists.